src/components/crds-site-happenings/site-happenings.tsx

```diff
@@ -278,10 +267,41 @@ export class SiteHappenings {
     this.host.shadowRoot.querySelector('.site-select-message').classList.add('hidden');
   }
 
+  /**
+   * Map 4 fake cards while processing data
+   * from ctfl
+   */
+  private renderHappeningsSkeleton() {
+    let arr = [];
+    if (window.innerWidth < 768) {
+      arr.push(1, 2);
+    } else if (window.innerWidth >= 768 && window.innerWidth < 960) {
```

Comment:
window.innerWidth >= 768 is implied because it passed the first conditional.

This should be sufficient
```suggestion
    } else if (window.innerWidth < 960) {
```

Reply:
@lukemoderwell Also, what is this solving? Is there a screen width issue or something? Why not just map 4 and let the other 1 or 2 be off screen?

Reply:
@j-brammer I want to have 2, 3, and 4 cards for different screen sizes to match the wrapping present when the component loads. If I have more it makes the skeleton larger resulting in possible jank when real content comes in.

Reply:
I also think you will hit a weird issue when you are at `768` which is standard iPad sizing. No if case currently will catch that.